### ISSUE TITLE
TwitterKit.Twitter.SharedInstance.Application Missing

### ISSUE THREAD

**Issue by jamesmundy** [joeisapro/MonoTouch.Fabric#4]
Hi Joe,

Great work on the library. I think something is missing though. On the Fabric Advanced Setup page (https://docs.fabric.io/apple/twitter/advanced-setup.html) it says to notify TwitterKit of OpenUrl like so:

```
func application(app: UIApplication, openURL url: NSURL, options: [String : AnyObject]) -> Bool {
    if Twitter.sharedInstance().application(app, openURL:url, options: options) {
        return true
    }

    // If you handle other (non Twitter Kit) URLs elsewhere in your app, return true. Otherwise
    return false
}
```

However, in the current binding I can find no reference to .Application in SharedInstance of Twitter. Any ideas?

**Comment by joeisapro** [comment]
Hi James,

After investigating, it appears this instance method was added to TwitterKit as of 2.2.0. 
The binding hasn't been updated to 2.2.0 yet - it's still on 1.9.0.  

The reason is that newer versions of Crashlytics don't play as nice with Xamarin.iOS as older ones.  I've worked out a lot of little issues, but still have more to address.  Once everything has been taken care of, I will be able to update all of the bindings.

Joe.
